packages/pwa-module/etc/module.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0"?>
-
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
     <module name="Magento_Pwa" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_Backend" />
+            <module name="Magento_Store" />
+        </sequence>
     </module>
 </config>
```

Comment:
Whilst we're here, let's add a new line to this file.

Reply:
Hmm, within PHP, newlines at the end of the file are part of the PSR standard. However, does this also apply to XML. It is simply the first time I heard so ...

Reply:
Internally we typically just add new lines to everything for consistency. Unsure if with XML it's part of any convention though.